neurom/analysis/morphtree.py

```diff
@@ -264,6 +264,40 @@ def trunk_length(tree):
         return 0.0
 
 
+def trunk_vector(tree, soma):
+    '''Vector of trunk defined as (initial tree point - soma center)
+       of the tree.
+    '''
+    return mm.vector(tree.value, soma.center)
+
+
+def trunk_elevation(tree, soma):
```

Comment:
Trunk should not mean first cross section. If this is what we think it means, then we need to use a word other than trunk. In fact, the work trunk makes almost no sense in this context.